opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -8625,7 +8625,7 @@ type Mutation {
   deleteImport(fileName: String): ID
   uploadImport(file: Upload!, fileMarkings: [String]): File
   uploadPending(file: Upload!, entityId: String, labels: [String], errorOnExisting: Boolean, refreshEntity: Boolean): File
-  askJobImport(fileName: ID!, connectorId: String, configuration: String, bypassEntityId: String, bypassValidation: Boolean): File
+  askJobImport(fileName: ID!, connectorId: String, configuration: String, bypassEntityId: String, bypassValidation: Boolean, validationMode: String): File
```

Comment:
I'm wondering if we shouldn't have an enum here instead of String ? so it can either be workbench or draft, not any string.